src/main.dev.js

```diff
@@ -248,6 +248,14 @@ app.on('ready', async () => {
     }
   });
 
+  // Ensure the application window has focus as well as the emmbedded content
+  // will be called on settings page and when url is switched to home url
+  mainWindow.webContents.on('dom-ready', () => {
+    console.log('DOM READY');
```

Comment:
Remove console.log

I assume that you're doing this here for dev purposes. If you think it's worth logging this state to the log file then make sure to use logger.info or some other state, and match the style of other log messages in the same file.